docs/source/primitives.rst

```diff
@@ -152,27 +156,27 @@ Schema
 Update
 ******
 
-Update tasks are used to change values in the context. The subtasks have three major forms:
+Update tasks are used to change values in the context. These tasks have three major forms:
```

Comment:
this needs fleshing out to capture the idea that an update task itself contains multiple tasks, of the three major forms - the thing they do is defined by the structure of the task (what keyword - maybe list this in someway?)